pyrostest/ros_test.py

```diff
@@ -49,9 +47,8 @@ def __init__(self, topic, msg_type, queue_size):
     def send(self, value):
         """Sends data to be publoshed to the mocked topic.
         """
+        time.sleep(.5)
```

Comment:
I'm thinking it could be an issue if you have to have multiple messages sent in a critical period of time, which if we are mocking message sending, then that's a very likely scenario to happen. (e.g.: say our median filter only keeps fixes that are 1sec old or younger).

But you said there's another fix in the works so i can approve this now and wait for that. I do think this is an issue tho